web/beacon-app/src/index.css

```diff
@@ -157,6 +157,18 @@
   min-height: calc(100vh - 600px);
 }
 
+.create-key-modal, .download-key-modal {
+  @media (min-width: 640px) {
+    width: min(50vw, 600px)!important;
+  }
+}
+
+.create-key-modal, .download-key-modal {
+  @media (max-width: 639px) {
```

Comment:
This style change instructs the browser to choose either a max-width of `80vw` or `300px` for screens with a max width of `639px`. !important is included to override styles set in the Beacon design system for all modals.